gym/core.py

```diff
@@ -482,11 +446,8 @@ def observation(self, obs):
 
     def reset(self, **kwargs):
```

Comment:
could we change this to the actual reset parameters?

Reply:
I'm not necessarily against this, but some our wrappers use this strategy to allow the wrapper to be agnostic to changes in the function definitions, such as https://github.com/openai/gym/blob/master/gym/wrappers/order_enforcing.py. Is this suggestion part of a broader goal of moving towards explicit argument type hinting for wrappers?

Reply:
That is a good. Lets not do it because it allows the wrappers to be partially backward compatible